test/e2e/suite/006-apps-in-any-namespace/99-delete.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: kuttl.dev/v1beta1
+kind: TestStep
+timeout: 120
+delete:
+- apiVersion: argoproj.io/v1alpha1
+  kind: Application
+  name: image-updater-006
+- apiVersion: v1
+  kind: Namespace
+  name: image-updater-e2e-006-01
+- apiVersion: v1
+  kind: Namespace
+  name: image-updater-e2e-006
```

Comment:
as part of the cleanup, we should probably also delete the AppProj project-one, and maybe also revert changes to `argocd-cmd-params-cm`.